index.js

```diff
@@ -32,15 +32,68 @@ PLAYERS.forEach(renderPlayer)
 
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
+const toggleColor = element => {
   if (element.style.color === "red") {
     element.style.color = "black"
   } else {
     element.style.color = "red"
   }
 }
 
+const header = document.querySelector("#header")
+header.addEventListener("click", () => {
+	toggleColor(header)
+})
+
 
 /***** Deliverable 2 *****/
 
+const newPlayer = document.querySelector("#new-player-form")
+
+
+newPlayer.addEventListener("submit", (event) => {
+	event.preventDefault()
+
+	// setting variables for input 
+	// 'number' refers to the input in the HTML form
+	const number = event.target.number.value
+	const name = event.target.name.value
+	const nickname = event.target.nickname.value
+	const photo = event.target.photo.value
+
+// using above variables to set new Player for rendering
+	const player = {
+		number: number,
+		name: name,
+		nickname: nickname,
+		photo: photo,
+		likes: 0
+	}
```

Comment:
Looks good! If you want, you can shorten this up a bit by declaring the object with the values directly:

```js
const player = {
    number: event.target.number.value,
    name: event.target.name.value,
    nickname: event.target.nickname.value,
    photo: event.target.photo.value,
    likes: 0
}